branding/src/test/scala/com/normation/plugins/branding/api/BrandingApiTest.scala

```diff
@@ -17,8 +17,6 @@ import org.specs2.specification.AfterAll
 class BrandingApiTest extends Specification with TraitTestApiFromYamlFiles with Loggable with AfterAll {
   sequential
 
-  val restTestSetUp = RestTestSetUp.newEnv
```

Comment:
this is rather heavy wait, in that case, I think it's unecessary to pull the whole rest test set up for the quick api test